bitnami/thanos/README.md

```diff
@@ -131,8 +131,16 @@ The following tables lists the configurable parameters of the Thanos chart and t
-| `querier.service.type`                          | Kubernetes service type                                                                                | `ClusterIP`                                             |
-| `querier.service.clusterIP`                     | Thanos Querier service clusterIP IP                                                                    | `None`                                                  |
+| `querier.grpcTLS.server.secure`                 | Enable TLS for GRPC server                                                                             | `false`                                                 |
+| `querier.grpcTLS.server.cert`                   | TLS Certificate for gRPC server                                                                        | `nil`                                                   |
+| `querier.grpcTLS.server.key`                    | TLS Key for gRPC server                                                                                | `nil`                                                   |
+| `querier.grpcTLS.server.ca`                     | TLS client CA for gRPC server used for client verification purposes on the server                      | `nil`                                                   |
+| `querier.grpcTLS.client.secure`                 | Use TLS when talking to the gRPC server                                                                | `false`                                                 |
+| `querier.grpcTLS.client.cert`                   | TLS Certificates to use to identify this client to the server                                          | `nil`                                                   |
+| `querier.grpcTLS.client.key`                    | TLS Key for the client's certificate                                                                   | `nil`                                                   |
+| `querier.grpcTLS.client.ca`                     | TLS CA Certificates to use to verify gRPC servers                                                      | `nil`                                                   |
+| `querier.grpcTLS.client.servername`             | Server name to verify the hostname on the returned gRPC certificates.                                  | `nil`                                                   |
+| `querier.service.type`                          | Kubernetes service type                                                                                | `ClusterIP`                                             || `querier.service.clusterIP`                     | Thanos Querier service clusterIP IP                                                                    | `None`                                                  |
```

Comment:
Be careful with this line, you deleted the new line before the `querier.service.clusterIP` parameter